jlm/llvm/ir/aggregation.cpp

```diff
@@ -55,25 +55,22 @@ AggregationNode::normalize(AggregationNode & node)
     normalize(child);
 }
 
-/* entryaggnode class */
+EntryAggregationNode::~EntryAggregationNode() noexcept = default;
```

Comment:
What does this do differently from what's already in the hope?

Reply:
I do not understand the question?

Reply:
@phate Håvard explained the benefit of adding this to the CPP and not only have it in the HPP.